### PR TITLE
support more controller manager params for helm charts

### PR DESCRIPTION
**Pull request by hantmac**
Signed-off-by: hantmac <hantmac@outlook.com>

- Fixes #<https://github.com/streamnative/function-mesh/issues/332>
- Make `enable-leader-election` default value as `true`.

### Motivation

Because of https://github.com/streamnative/function-mesh/pull/329, we should support more controller manager params for helm charts.

### Verifying this change

- [x] Make sure that the change passes the CI checks.

### Documentation

Need to update docs? 

  
- [x] `no-need-doc`

### CODE REVIEW

**Comment by hantmac** [comment]
@freeznet Do I need to update the related docs? And if need, which doc repo?

**Comment by freeznet** [comment]
ping @Huanli-Meng, could you please help @hantmac? thanks.

File: charts/function-mesh-operator/values.yaml [hunk 1]
```diff
@@ -59,3 +59,12 @@ controllerManager:
   # resourceLabels: {}
   # resource annotations applied to each function/connector managed by this controller
   # resourceAnnotations: {}
+  webhook:
+    configFile: /etc/config/config.yaml
```
Comment (freeznet) [inline charts/function-mesh-operator/values.yaml:63]: the configFile is not for webhook, please check https://github.com/streamnative/function-mesh/pull/322 for more info.

File: charts/function-mesh-operator/templates/controller-manager-deployment.yaml [hunk 2]
```diff
@@ -42,7 +42,11 @@ spec:
           - /manager
         args:
           - --enable-leader-election
-          - --config-file=/etc/config/config.yaml
+          - --enable-pprof
```
Comment (freeznet) [inline charts/function-mesh-operator/templates/controller-manager-deployment.yaml:45]: I think `pprof` should be disabled by default

**Review by freeznet — CHANGES_REQUESTED** [review]

**Comment by freeznet** [comment]
@hantmac please rebase master and check my comment, thanks.

**Comment by Huanli-Meng** [comment]
> ping @Huanli-Meng, could you please help @hantmac? thanks.

OK. By the way, do we need to add doc for this PR. I noticed that a `doc-info-missing` label is added

File: charts/function-mesh-operator/templates/controller-manager-deployment.yaml [hunk 3]
```diff
@@ -42,7 +42,11 @@ spec:
           - /manager
         args:
           - --enable-leader-election
-          - --config-file=/etc/config/configs.yaml
+          - --enable-pprof
```
Comment (freeznet) [inline charts/function-mesh-operator/templates/controller-manager-deployment.yaml:45]: `--enable-pprof` havnt refer to the Values, we should add `--enable-pprof` only if `.Values.controllerManager.enable-pprof` is true.

File: charts/function-mesh-operator/values.yaml [hunk 4]
```diff
@@ -59,3 +59,13 @@ controllerManager:
...
   # resourceAnnotations: {}
+
+  configFile: /etc/config/config.yaml
+  enable-leader-election: true
+  enable-pprof: false
+  metrics:
+    port: 8080
+  healthProbe:
+    port: 8000
+  pprof:
```
Comment (freeznet) [inline charts/function-mesh-operator/values.yaml:70]: how about move `enable-pprof` to `pprof` section, like:

```
pprof:
  enable: false
  port: 8090
```

**Review by freeznet — COMMENTED** [review]
please check my comment, thanks.

**Comment by freeznet** [comment]
@hantmac the docs can be updated at https://github.com/streamnative/function-mesh-website
@Huanli-Meng https://github.com/streamnative/function-mesh/pull/329 added some params to the controller, and user can customize them for particular purpose.

File: charts/function-mesh-operator/templates/controller-manager-deployment.yaml (hunk 3 above)
Comment (hantmac) [reply charts/function-mesh-operator/templates/controller-manager-deployment.yaml]: sorry, I was careless. Fixed.

File: charts/function-mesh-operator/values.yaml (hunk 4 above)
Comment (hantmac) [reply charts/function-mesh-operator/values.yaml:69]: agree

**Comment by hantmac** [comment]
> @hantmac the docs can be updated at https://github.com/streamnative/function-mesh-website @Huanli-Meng #329 added some params to the controller, and user can customize them for particular purpose.

Copy that.

**Review by freeznet — APPROVED** [review]

**Pull request merged**